pages/doc/proxies_configuring.md

```diff
@@ -220,8 +221,8 @@ Properties specific to histogram distributions, listed in a <a href="#histogram-
 </tr>
 <tr>
 <td>idFile</td>
-<td>Location of the PID file for the wavefront-proxy process. <br/>Default: &lt;cfg_path&gt;/.wf_id.</td>
-<td>Valid path on the local file system.</td>
+<td>Location of the PID file for the wavefront-proxy process. <br/>Default: &lt;dshell&gt;/.id</td>
+<td>Valid path on the local file system. This configuration is ignored when ephemeral=true.</td>
```

Comment:
This configuration is ignored when ephemeral=true. 
Maybe: This option is ignored when ephemeral=true. 
?